sentry/src/main/java/io/sentry/util/Pair.java

```diff
@@ -0,0 +1,39 @@
+package io.sentry.util;
+
+import org.jetbrains.annotations.Nullable;
+
+public final class Pair<A, B> {
+  private final @Nullable A first;
+  private final @Nullable B second;
+
+  public Pair(final @Nullable A first, final @Nullable B second) {
+    this.first = first;
+    this.second = second;
+  }
+
+  public @Nullable A getFirst() {
+    return first;
+  }
+
+  public @Nullable B getSecond() {
+    return second;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+
+    Pair<?, ?> pair = (Pair<?, ?>) o;
+
+    if (first != null ? !first.equals(pair.first) : pair.first != null) return false;
+    return second != null ? second.equals(pair.second) : pair.second == null;
+  }
+
+  @Override
+  public int hashCode() {
+    int result = first != null ? first.hashCode() : 0;
+    result = 31 * result + (second != null ? second.hashCode() : 0);
+    return result;
+  }
```

Comment:
do we need that if it's just a container?

Reply:
Simplifies equality check in tests

Reply:
true but better asserting field per field in tests, we don't want to bloat the SDK with stuff used only in tests